qiita_db/software.py

```diff
@@ -455,6 +482,38 @@ def add_publications(self, publications):
             qdb.sql_connection.TRN.add(sql, sql_params, many=True)
             qdb.sql_connection.TRN.execute()
 
+    @property
+    def environment_script(self):
+        """The script used to start the plugin environment
+
+        Returns
+        -------
+        str
+            The script used to start the environment
```

Comment:
I see, makes sense, thanks! Would be useful to note that this refers to
a command available within a non-interactive session.

On (Dec-04-15|13:32), Jose Navas wrote:
> > @@ -455,6 +482,38 @@ def add_publications(self, publications):
> >              qdb.sql_connection.TRN.add(sql, sql_params, many=True)
> >              qdb.sql_connection.TRN.execute()
> >  
> > +    @property
> > +    def environment_script(self):
> > +        """The script used to start the plugin environment
> > +
> > +        Returns
> > +        -------
> > +        str
> > +            The script used to start the environment
> 
> Sorry, I got confused here. This is the call used to start the environment. So it can be either a script, or whatever. For example, in my system this looks like `workon qiita`; in the test environment this looks like `source activate qiita` (using conda) or I can have a `start_plugin_X_env.sh` which is a bash script. At the end is a CLI call that I can issue to start the environment
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1558/files#r46734863